src/features/components/organisms/options-top-tab/data.ts

```diff
@@ -12,21 +12,23 @@ export const tab_data: OptionNavigationType[] = [
         src: DigitalnActive,
         selected_src: DigitalActive,
         button_text: '_t_Digital options_t_',
-        active_path: ['/trade-types/options/digital-options/'],
+        active_path: '/trade-types/options/digital-options/',
         option_name: 'digital',
         to: '/trade-types/options/digital-options/up-and-down',
     },
     {
         src: AccumulatorslInActive,
         selected_src: AccumulatorslActive,
         button_text: '_t_Accumulator options_t_',
+        active_path: '/trade-types/options/digital-options/accumulator-options',
```

Comment:
Why it is **/digital-options/accumulator-options**. Shouldn't it be **/trade-types/options/accumulator-options**